datadog_checks_dev/datadog_checks/dev/tooling/commands/validate/licenses.py

```diff
@@ -14,10 +16,71 @@
 
 from ....fs import file_exists, read_file_lines, write_file_lines
 from ...constants import get_agent_requirements, get_license_attribution_file
+from ...github import get_auth_info
 from ...utils import get_extra_license_files, read_license_file_rows
 from ..console import CONTEXT_SETTINGS, abort, annotate_error, echo_failure, echo_info, echo_success
 
+# Files searched for COPYRIGHT_RE
+COPYRIGHT_LOCATIONS = [
+    'license',
+    'LICENSE',
+    'license.md',
+    'LICENSE.md',
+    'LICENSE.txt',
+    'License.txt',
+    'license.txt',
+    'COPYING',
+    'NOTICE',
+    'README',
+    'README.md',
+    'README.mdown',
+    'README.markdown',
+    'COPYRIGHT',
+    'COPYRIGHT.txt',
+    'COPYING.txt',
+    'LICENSE-APACHE',
+    'LICENSE.APACHE',
+    'LICENSE.BSD',
+    'LICENSE.PSF',
+    'LICENSE-MIT',
+]
+
+
+# General match for anything that looks like a copyright declaration
+COPYRIGHT_RE = re.compile(
```

Comment:
These compilations will have an impact on start-up time. Can we move these to a separate file and import them when necessary or, since Python caches the most recent 256 compiled regular expressions, you could just use the unbound methods.